README.md

```diff
@@ -36,6 +55,28 @@ network and run:
  clojure -A:test integration
 ```
 
+For end-to-end testing, you need to connect to the VPN or be in the Broad
+network and run:
+```bash
+ clojure -A:test e2e
+```
+
+The e2e test runs in the dev environment by default. To run in production:
+```bash
+ZAMBONI_ACTIVEMQ_SERVER_URL="failover:ssl://vpicard-jms.broadinstitute.org:61616"
+ZAMBONI_ACTIVEMQ_QUEUE_NAME="wfl.broad.pushtocloud.enqueue.prod-test"
+ZAMBONI_ACTIVEMQ_SECRET_PATH="secret/dsde/gotc/prod/activemq/logins/zamboni"
+PTC_BUCKET_URL="gs://broad-aou-arrays-input"
+CROMWELL_URL="https://cromwell-aou.gotc-prod.broadinstitute.org"
+WFL_URL="https://aou-wfl.gotc-prod.broadinstitute.org"
+
+ZAMBONI_ACTIVEMQ_SERVER_URL=${ZAMBONI_ACTIVEMQ_SERVER_URL} \
```

Comment:
You can just remove this by turning the above into `export`s

Reply:
Good point!